azext_iot/central/_help.py

```diff
@@ -1375,6 +1650,232 @@ def _load_central_edge_help():
+    """
+
+    helps[
+        "iot central device edge module properties update"
+    ] = """
+        type: command
+        short-summary: Update property values of a module.
+        examples:
+        - name: Update module properties via patch.
+          text: >
+            az iot central device edge module properties update
+            --app-id {appid}
+            --device-id {deviceId}
+            --module-name {moduleName}
+            --content {json}
+    """
+
+    helps[
+        "iot central device edge module properties replace"
+    ] = """
+        type: command
+        short-summary: Replace all property values of a module.
+        examples:
+        - name: Replace module properties.
+          text: >
+            az iot central device edge module properties replace
+            --app-id {appid}
+            --device-id {deviceId}
+            --module-name {moduleName}
+            --content {json}
+    """
+
+    helps[
+        "iot central device edge module show-telemetry-value"
+    ] = """
+        type: command
+        short-summary: Get the last telemetry value from a module.
+        examples:
+        - name: Get module telemetry value.
+          text: >
+            az iot central device edge module show-telemetry-value
+            --app-id {appid}
+            --device-id {deviceId}
+            --module-name {moduleName}
+            --telemetry-name {telemetryName}
+    """
+
+    helps[
+        "iot central device edge module command"
+    ] = """
+        type: group
+        short-summary: Run module commands.
+    """
+
+    helps[
+        "iot central device edge module command run"
+    ] = """
+            type: command
+            short-summary: Run a module command.
+            long-summary: |
+                Note: payload should be nested under "request".
+                i.e. if your device expects the payload in a shape {"key": "value"}
+                payload should be {"request": {"key": "value"}}.
+                --content can also be pointed at a filepath like this (.../path/to/payload.json)
+            examples:
+            - name: Run a module command
+              text: >
+                az iot central device edge module command run
+                --app-id {appid}
+                --device-id {deviceid}
+                --module-name {modulename}
+                --command-name {commandname}
+                --content {payload}
+        """
+
+    helps[
+        "iot central device edge module command run-component-command"
+    ] = """
+            type: command
+            short-summary: Run a module component command.
+            long-summary: |
+                Note: payload should be nested under "request".
+                i.e. if your device expects the payload in a shape {"key": "value"}
+                payload should be {"request": {"key": "value"}}.
+                --content can also be pointed at a filepath like this (.../path/to/payload.json)
```

Comment:
content note should be part of the param (inline content or path to filepath with content) and a separate example(s). Add one for file path and one for the payload example. 

will content ever have anything aside from the payload?